### PR TITLE
Galileo/Minnowboard documentation updates

### PR DESCRIPTION
**Pull request by nashif**
- info about minnowboard firmware and how to create a boot image
- cleanup galileo docs a little bit and reused for MinnowBoard.

### CODE REVIEW

File: boards/x86/galileo/doc/galileo.rst [hunk 1]
```diff
@@ -217,10 +218,10 @@ GRUB.
    .. code-block:: console
 
      $ cd $ZEPHYR_BASE
-     $ ./boards/x86/galileo/support/build_grub.sh
+     $ ./boards/common/scripts/build_grub.sh
 
 #. Find the binary at
-   :file:`$ZEPHYR_BASE/./boards/x86/galileo/support/grub/bin/grub.efi`.
+   :file:`$ZEPHYR_BASE/./boards/common/scripts/grub/bin/grub.efi`.
```
Comment (dbkinder) [inline boards/x86/galileo/doc/galileo.rst]: Do you need the ``/.`` in this path?

File: boards/x86/minnowboard/doc/minnowboard.rst [hunk 2]
```diff
@@ -66,4 +79,19 @@ The following platform features are unsupported:
...
+
+Booting Zephyr on the Minnowboard
+=================================
+
+The MinnowBoard by default will be running a 64bit firmware. To boot Zephyr, you
+will need to replace the 64bit firmware with the 32bit variant. Please follow
+the instructions for flashing and updating the firmware documented at the
+`MinnowBoard`_ website.
+
+
```
Comment (dbkinder) [inline boards/x86/minnowboard/doc/minnowboard.rst]: This URL would get folks to the firmware update instructions:  https://minnowboard.org/tutorials/updating-the-firmware

**Review by dbkinder — CHANGES_REQUESTED** [review]
Might be a good idea to mention the Galileo boards were discontinued by Intel in June 2017.  (https://software.intel.com/en-us/iot/hardware/discontinued)

File: boards/x86/galileo/doc/galileo.rst [hunk 3]
```diff
@@ -190,12 +190,13 @@ Use the following procedures for booting an image on a Galileo board.
...
+.. _grub2:
+
 Creating a GRUB2 Boot Loader Image from a Linux Host
 ====================================================
 
-If you are having problems running an application using the default GRUB
-of the hardware, follow these steps to test on Galileo2 boards using a custom
-GRUB.
+If you are having problems running an application using the default GRUB of the
+hardware, follow these steps to test on supported boards using a custom GRUB.
```
Comment (andrewboie) [inline boards/x86/galileo/doc/galileo.rst]: little awkward wording. maybe "the preinstalled copy of GRUB" or something.

File: boards/x86/galileo/support/build_grub.sh [hunk 4]
```diff
@@ -1,64 +0,0 @@
-#!/bin/bash
-
```
Comment (andrewboie) [inline boards/x86/galileo/support/build_grub.sh]: it looks like this script was deleted and not moved?

**Pull request merged**